vault/auth_token.go

```diff
@@ -117,7 +118,7 @@ func addTokenFields(fields map[string]*schema.Schema, config *addTokenFieldsConf
 	}
 }
 
-func setTokenFields(d *schema.ResourceData, data map[string]interface{}, config *addTokenFieldsConfig) {
+func addRequestTokenFields(d *schema.ResourceData, data map[string]interface{}, config *addTokenFieldsConfig) {
```

Comment:
I noticed that after renaming, we're not using this function anymore. Is it safe to remove this?

Reply:
Yeah, that was not the idea... Reverting.

Reply:
I meant to hold that change back, but part of it slipped through.

Reply:
Ah I think I uncovered a bug, we would also need to update line 120 of `resource_token_auth_backend_role.go` where `setTokenFields` is being called instead of `addRequestTokenFields`. That also causes the tests to fail.

Tests are green once that line is fixed though! :)

Reply:
Fixed in https://github.com/hashicorp/terraform-provider-vault/pull/1346/commits/baaa0f8bd6315462c4bacc5c455bf2232fabc2f7

Reply:
I reverted it in the mean time, since it was rather unrelated to this fix.

Reply:
Nice, thanks!